packages/headless/src/features/generated-answer/generated-answer-state.ts

```diff
@@ -82,6 +82,10 @@ export interface GeneratedAnswerState {
    * Whether the answer is expanded.
    */
   expanded: boolean;
+  /**
+   * Determines if no answer is generated after a query is executed.
+   */
+  hasNoAnswer: boolean;
```

Comment:
I agree with @dmgauthier , I think something like this would make more sense.
```suggestion
  /**
   * Whether an answer cannot be generated after a query is executed.
   */
  cannotAnswer: boolean;
```